### ISSUE TITLE
Add Python.Execute() function to Excel

### ISSUE THREAD

**Issue by cbaragao** [microsoft/python-in-excel#33]
I would love to see Python.Execute() in Power Query for Excel. It would be a great way extend transformation and data cleaning capabilities in Power Query for Excel.

**Comment by keyur32** [comment]
Thanks for this request. To help me understand, can you outline what you're looking for with the Python.Execute() method? Are you asking if you can run a python formula directly as part of a pq transformation?

**Comment by ncalm** [comment]
I guess bring parity between Power Query in PBI and Power Query in Excel. 
https://learn.microsoft.com/en-us/power-bi/connect-data/desktop-python-in-query-editor

**Comment by cbaragao** [comment]
Yeah, basically to be able to call Python from PowerQuery in Excel like you can with Power BI. Power Query in Power BI has the Python.Execute() function, Power Query in Excel does not.

**Comment by icepero** [comment]
I second this request. I can call a REST api easily from Power BI but I would like to do it in PowerQuery in Excel.

**Comment by rootsmusic** [comment]
@keyur32 Since you have Python in Excel, why isn't Python script supported in Excel's Power Query?
